samples/fullduplex/Core_10/Shared/RequestDataMessage.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+using NServiceBus;
+
+#region RequestMessage
+// C# 12+ Primary constructor with record for immutable message
+public record RequestDataMessage(Guid DataId, string String) : IMessage;
```

Comment:
If you plan to switch to using record, the sample instructions also need to be updated.